### DISCUSSION TITLE
4k/60p HEVC File becomes 59.940fps after Cut and Join process

### DISCUSSION THREAD

**Discussion by glenpinn** [mifi/lossless-cut#2198] [Support]
Hi there, i now record videos in 4k/60p HEVC mode at either 60Mbps or 144Mbps, and because my VideoReDo Pro software is jittery and lags in the timeline when i preview or seek to my cutting points, i am now using LosslessCut to do my cutting and joining as it seems to handle this process very smoothly and fast.

I have a question, when i do certain types of cuts and join the files together, some of my output files show up in MediaInfo as 60.0fps just as the source file is, but others show as being 59.940fps.

I have uploaded several screenshots of various cuts that i have tried, indicating the resulting frame rates on each file.

Can you please explain why this might happen, and which of the screenshot files should i be using when cutting a file into 2 or more segments and joining it together, should i do all my cuts only on keyframes, including the start and end frames too.

Cheers.

![01 S23 4k @ 60fps](https://github.com/user-attachments/assets/f417d1b0-1eae-4be6-8c67-c2861fcf4408)
![02 S23 4k @ 60fps](https://github.com/user-attachments/assets/89804c9d-28d7-404d-92b4-eabce03a89eb)
![03 S23 4k @ 59 940fps](https://github.com/user-attachments/assets/be2cde05-5484-4af5-a345-e98537dca2a1)
![04 S23 4k @ 59 940fps](https://github.com/user-attachments/assets/b1f9cb11-8108-41c7-a02d-f937ed0fa95e)
![05 S23 4k @ 59 940fps](https://github.com/user-attachments/assets/fab06f6e-09a9-4258-8797-331dccc0425f)
![06 S23 4k @ 60fps](https://github.com/user-attachments/assets/ab12a261-0d3c-4c42-acc4-0653b2756db8)
![07 S23 4k @ 60fps](https://github.com/user-attachments/assets/4bb989a4-4d89-4364-9f54-b24ce105976f)
![08 S23 4k @ 59 940fps](https://github.com/user-attachments/assets/921ef6ab-dbe2-4a4b-8a7c-5121f3cf5a12)
![09 S23 4k @ 59 940fps](https://github.com/user-attachments/assets/1584f440-2330-4d7d-a6b1-73aec93ac452)

**Comment by mifi** [comment]
Hi! I've seen similar issues but unfortunately I don't have an explanation. Related #1027
